### PR TITLE
Feature/issues/185

### PR DESCRIPTION
**Pull request by Mizzick**

### CODE REVIEW

File: Extension/lib/content-script/assistant/js/adguard-selector.js [hunk 1]
```diff
@@ -417,44 +481,87 @@ var AdguardSelectorLib = (function (api, $) {
...
 
     var setupEventHandlers = function () {
         makeIFrameAndEmbededSelector();
 
         var sgIgnore = $("body *:not(." + IGNORED_CLASS + ")");
-        sgIgnore.on("mouseover", sgMouseoverHandler);
-        sgIgnore.on("mouseout", sgMouseoutHandler);
-        sgIgnore.on("click", sgMousedownHandler);
+
+        if (isIOS) {
```
Comment (Mizzick) [inline Extension/lib/content-script/assistant/js/adguard-selector.js]: @Stillness-2 is it possible to do it without this flag?
We can do it more generic, touch events could be useful not only for ios.
Or common events(mouseover, mouseout, click) don't work in ios?

Comment (ameshkov) [inline Extension/lib/content-script/assistant/js/adguard-selector.js]: Even if it is not possible, we should not make an "iOS" exception.

There are lots of touch devices out there, surface pro for instance.

**Comment by ameshkov** [comment]
Seems to be nice. Have you checked how it works?

**Comment by Mizzick** [comment]
It's fine on desktop, but haven't check with touch devices, waiting for @Stillness-2 to do it.

**Pull request merged**